### ISSUE TITLE
ada accessibility

### ISSUE THREAD

**Issue by lewislwood** [neondatabase/neon#6020]
Website is lacking accessibility.

First of all barely any regions in dashboard screens.
No headings on dashboard screens.

Some table columns where you display branches and databases, these columns have unlabeled buttons (alt-desc) or table heading in tsh is needed.

Connection the 2 buttons I had to just press and hope not self destruct.
One buton is expose passwords. Other was copy.  They both could use alt-desc .
I barely managed. Following a tutorial Dave Gray Nest lesson05. What took him 1 minutes, took me more than hour to figure it out. 

Simple accessibilty will make this a good experience for all, especially me. If you had done accessibilty I would habe gotten done in 30 seconds, since he talked a lot.

**Comment by ololobus** [comment]
cc @stepashka @seymourisdead @PlayLikeNeverB4

**Comment by seymourisdead** [comment]
https://github.com/neondatabase/cloud/issues/9095
